solidity/ecdsa/deploy/10_deploy_proxy_admin_with_deputy.ts

```diff
@@ -0,0 +1,27 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+import type { ProxyAdmin } from "../typechain"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { ethers, getNamedAccounts, upgrades, deployments } = hre
+  const { esdm } = await getNamedAccounts()
+
+  const ProxyAdminWithDeputy = await deployments.get("ProxyAdminWithDeputy")
```

Comment:
I am confused. Is this script deploying or trying to fetch an already deployed instance? The script is named `10_deploy*` so I'd expect to see `await deployments.deploy` instead of `awaits deployments.get`.

Reply:
Good catch, this had to be a mistake.

Reply:
5bfc5285996d172a5bb03f1cc2beb91eda60f163